### PR TITLE
Remove milestoneType and partyRole codelists

### PR DESCRIPTION
**Pull request by ColinMaudry**
PR for the addition to the standard: https://github.com/open-contracting/standard/pull/1243

Related issue: https://github.com/open-contracting/standard/issues/1157

### CODE REVIEW

**Comment by ColinMaudry** [comment]
'notary' may stay. Maybe.

**Comment by ColinMaudry** [comment]
This PR depends on the review of open-contracting/standard#1243.

File: README.md [hunk 1]
```diff
@@ -78,6 +78,10 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### 2021-03-11
+
+* Move the codes from `+milestoneType.csv` and `+partyRole.csv` to the standard [#1157](https://github.com/open-contracting/standard/issues/1157).
```
Comment (jpmckinney) [inline README.md:83]: Use same format as other extensions.

```suggestion
* [#1157](https://github.com/open-contracting/standard/issues/1157) Move the codes from the +milestoneType.csv and +partyRole.csv codelists to the standard.
```

**Comment by jpmckinney** [comment]
@ColinMaudry I think this can be moved out of draft?

**Comment by ColinMaudry** [comment]
@jpmckinney  Several codes didn"t make it to the standard, they should consequently be readded to the PPP extension (eg https://github.com/open-contracting/standard/pull/1243#discussion_r616236685)

- partyRole.notary
- partyRole.socialWitness
- partyRole.otherWitness
- partyRole.grantor
- partyRole.consortiaMember

Or deleted?

**Comment by jpmckinney** [comment]
They don't need to be re-added, because the OCDS for PPPs profile doesn't use these codes. It's okay to just delete them.

**Comment by jpmckinney** [comment]
I'll make commits separately due to conflicts.

**Pull request closed without merging**